src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -468,6 +485,8 @@ export const BorrowInput = ({
           assetSymbol={collateralAsset.symbol}
           assetIcon={collateralAsset.icon}
           onChange={handleDepositInputChange}
+          handleIsInputtingFiatSellAmountChange={handleToggleCollateralAssetIsFiat}
+          isInputtingFiatSellAmount={collateralAssetIsFiat}
```

Comment:
This looks odd but is correct, the implementation of `TradeAssetInput` only toggles. Happy to leave this as is but we'll eventually need to change the prop `handleIsInputtingFiatSellAmountChange` inside `TradeAssetInput` to `onToggleIsInputtingFiatSellAmount` or similar such that consumers of that component arent aware of the inner workings of it (and therefore prone to breaking should the implementation change)

Reply:
43d2b22e2eae3191332e33f419b282bcb16050db